src/bootstrap/config.rs

```diff
@@ -204,6 +200,7 @@ pub struct Config {
     pub backtrace_on_ice: bool,
 
     // llvm codegen options
+    pub llvm_from_ci: bool,
```

Comment:
```suggestion
    /// Separate from `LLVMConfig` to avoid invalidating the cache - the .so object should be same whether it was downloaded or built from source
    pub llvm_from_ci: bool,
```